torch/_inductor/codecache.py

```diff
@@ -807,6 +800,7 @@ def compiled_fx_graph_hash(
 
 
 def cudagraph_post_compile(
+    gm: torch.fx.GraphModule,
```

Comment:
Just confirming, are we sure the constants on the gm always available post dynamo, and won't change during autograd or inductor compile? As long as they're avaliable on the dynamo generated forward graph for both the forward and the backward, we can use it in AOTAutogradCache.

Are the constants the same for the forward *and* backward graphs? Since we'll be passing the same ones to both? 

Either way, I think instead of passing in the netire gm to cudagraph_post_compile, I would prefer if you passed only the constants in a list as an argument, and moved the get_constants call to be at the caller of the function.